src/utils.ts

```diff
@@ -7,3 +7,16 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export const getVisisibleItems = (
```

Comment:
Here I understand a small typo, be careful with such mistakes. It should be `getVisibleItems` instead of `getVisisibleItems`.